pkg/app/piped/apistore/eventstore/store.go

```diff
@@ -142,13 +142,16 @@ func (s *store) Getter() Getter {
 	return s
 }
 
-func (s *store) GetLatest(ctx context.Context, name string, labels map[string]string) (*model.Event, bool) {
+func (s *store) GetLatest(ctx context.Context, name string, labels map[string]string, forceRefresh bool) (event *model.Event, cacheUsed bool, ok bool) {
```

Comment:
`noUsingCache` seems to more suitable in this case, but what about?

Reply:
I feel you, that looks neat because of readability within `GetLatest()`. But one thing that I'd like to emphasize is refreshing the cache which will be used next time as well.

Reply:
That being said, if you think about it, `notUsingCache` is enough in this case. Thank you! let me apply it.

Reply:
Applied at 5bf7e27